StRoot/StFstHitMaker/StFstHitMaker.h

```diff
@@ -0,0 +1,37 @@
+#ifndef StFstHitMaker_hh
+#define StFstHitMaker_hh
+
+#include "StMaker.h"
+
+class THashList;
+
+
+/**
+ * Calculates hit global position, and writes FST hits to StFstHitCollection.
+ *
+ * \author: Shenghui Zhang
+ * \date Oct. 2021
+ */
+class StFstHitMaker : public StMaker
+{
+public:
+
+   StFstHitMaker( const char *name = "fst_hit" );
+   Int_t InitRun(Int_t runnumber);
+   Int_t Make();
+
+protected:
+
+   THashList *mSensorTransforms; ///< A list of TGeo transformations for each FST sensor
+
+   ClassDef(StFstHitMaker, 0);
+};
+
+#endif
+
```

Comment:
Just a question : How are versions beeing kept track of within git. There must be something equivalent to $Id... as in cvs

Reply:
There is no automatic versioning for individual files in git. A unique hash generated for each update (i.e. commit) is used to identify the state of the entire repository at any given time.

Reply:
There is a similar feature called `export-subst` that allows to make substitutes in `git-archive`. But please don't use that one, in certain cases it can lead to generation of irreproducible source tarballs. We could instead have cons query git to see the current status of the local repo and populate a preprocessor define.